examples/wasm-example/src/main.rs

```diff
@@ -0,0 +1,152 @@
+//! This is a small WASM app using the Yew UI framework showcasing how to use Subxt's features in a WASM environment.
+//!
+//! To run the app locally use Trunk, a WASM bundler:
+//! ```
+//! cargo install --locked trunk
+//! ```
+//! Run the app locally:
+//! ```
+//! trunk serve --open
```

Comment:
After running this command on the repo `trunk serve --open` the `Cargo.lock` will get modified leading to some git uncommitted changes. Could we make sure that the `Cargo.lock` is up to date before merging this? Thanks!